portfolio/src/main/webapp/index.html

```diff
@@ -24,6 +24,15 @@ <h1>Farid Salau | Portfolio</h1>
       <div id="greeting-container"></div>
       <a target="_blank" style="margin:0;" href="https://www.linkedin.com/in/faridsalau/">Let's Connect!</a>
       <a target="_blank" href="https://github.com/faridsalau/">GitHub</a>
+      <form action="/data" method="POST">
+        <p>Add a comment:</p>
+            <textarea class="textarea-field" name="comment" id="comment-area"></textarea>
+        <p id="error-message"></p>
+        <button id="submit-btn" type="submit">Submit comment</button>
+      </form>
+      <div id="comments">
+          <h2 id="comments-title"></h2>
```

Comment:
Great `id`s here. It really makes me, the reader, know what's going on even though I didn't code this 💯